JSDemos/Demos/DataGrid/RowTemplate/description.md

```diff
@@ -1 +1,3 @@
-The **DataGrid** widget supports custom row templates that allow you to&nbsp;place virtually anything into a&nbsp;grid row. A&nbsp;custom row template is&nbsp;assigned to&nbsp;the **rowTemplate** option.
\ No newline at end of file
+**DataGrid** supports custom row templates. Use the  [rowTemplate](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#rowTemplate) ([rowRender](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#rowRender) in React) option to configure a template for rows.
+
```

Comment:
Row templates have some specificities, such as they should be declared as a `<tbody>` element with the `dx-row` class. Besides that, row templates have many restrictions, which are listed in the `rowTemplate` description. We should mention all this in this description.